browser/ui/webui/brave_new_tab_page_refresh/brave_new_tab_page.mojom

```diff
@@ -193,6 +193,9 @@ interface NewTabPageHandler {
   // Reverses the most recent action on the custom top sites list.
   UndoCustomTopSiteAction() => ();
 
+  // Notifies the browser that the user has clicked a top site.
+  MarkEntryPointAsTopSite(TopSite top_site);
```

Comment:
From the point of view of the NTP, this name is a bit difficult to understand. I suggest the more obvious:

```
ReportTopSiteClicked();
```

or

```
NotifyTopSiteClicked();
```

Since we aren't currently using the `top_site` parameter, let's remove that as well. We can add the parameter later if needed.